ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -535,31 +531,39 @@ const CompletionBanner = ({
           profile.initialCreditsExpirationEpochMillis && (
             <>
               <div style={styles.completedText}>
-                Your credits expire on{' '}
+                You have $
+                {initialCreditsUsage
+                  ? initialCreditsLimit - initialCreditsUsage
```

Comment:
This display needs a `formatInitialCreditsUSD()` wrapper.  As seen for my user:
<img width="627" alt="Screenshot 2025-05-21 at 1 46 35 PM" src="https://github.com/user-attachments/assets/b914fdb6-ab5f-4328-a73f-e70612feb838" />